src/chains/tests/test_signals.py

```diff
@@ -91,3 +92,91 @@ def test_on_chain_update_with_no_flush_token_set(self) -> None:
         ChainFactory.create()
 
         assert len(responses.calls) == 0
+
+
+@override_settings(
+    CGW_URL="http://127.0.0.1",
+    CGW_FLUSH_TOKEN="example-token",
+)
+class FeatureHookTestCase(TestCase):
+    @responses.activate
+    def test_on_feature_create_hook_call(self) -> None:
+        responses.add(
+            responses.POST,
+            "http://127.0.0.1/v1/flush/example-token",
+            status=200,
+            match=[responses.matchers.json_params_matcher({"invalidate": "Chains"})],  # type: ignore[no-untyped-call]
+        )
+
+        Feature(key="Test Feature").save()
```

Comment:
@Uxio0 the `@factory.post_generation` from `FeatureFactory` seems to create and update the feature in one go (probably due to setting the M2M relationship after creation).

So `FeatureFactory.create()` results in two signals while `Feature(key="Test Feature").save()` results in one (as expected).

Reply:
Yes, I've seen that before 😉 . One thing to take into consideration is that `bulk` requests don't trigger signals